### DISCUSSION TITLE
Not ChatGPT

### DISCUSSION THREAD

**Discussion by kaanalp28** [Zero6992/chatGPT-discord-bot#104] [General]
After struggling all day making a bot with GPT-3.5: text-davinci-003, (which i thought was chatGPT), after I started to see notable differences between my bot and ChatGPT (the actual website), I decided to research and found out that these are not the same. 

I honestly didn't needed a more powerful bot, noone in discord is gonna make huge dataset adjustments or those kinds of things (which davinci is better at than chatGPT), what I needed was a good bot that was human like.

When OpenAI was making chatgpt, they adjusted it to make more conversation friendly and human like. That's what I wanted to get, since in discord, everyone would rather chat with a thing that can keep track of things easier. (Easier to communicate in general.)

Thats why I started researching, to find the api of ChatGPt (NOT text-davinci-003). I thought this was it, but after looking at some files, it turned out to be -again- text-davinci-003. I'm sad :(

If anyone knows how can I get the api for ChatGPT (NOT davinci etc.), please lemme know I'm tired of searching the internet.

(I filled out my hourly question limit tryna find out)

Discuss in the comments! :)

**Comment by Ammy076** [comment]
How to Create an API
Determine Your Requirements. First, you'll need to determine your API requirements. ...
Design Your API. Next, you'll need to consider API design. ...
Develop Your API. Now, it's time to start developing your API. ...
Test Your API. ...
Publish/Deploy Your API. ...
Monitor Your API.
it will help you

**Reply by Zero6992** [reply]
I will update ASAP

**Comment by Zero6992** [comment]
Updated to ChatGPT API now
